schema/build/generate_schema.py

```diff
@@ -5,6 +5,7 @@
 import json
 from pathlib import Path
 import re
+from webbrowser import get
```

Comment:
I don't believe this is used. Can this be removed?

Reply:
Removes